tests/transports-governance/test_governance_hierarchical_models.py

```
@@ -0,0 +1,908 @@
+- RateLimit operations and enforcement rules
+- Customer operations and budget relationships
+- Team operations and customer associations
+- VirtualKey operations and mutual exclusion constraints
+- Model relationships and hierarchy validation
+- Data validation and constraint enforcement
+- Error handling and edge cases
+"""
+
+import pytest
+import uuid
+import time
```

Ruff (0.12.2)

`time` imported but unused

Remove unused import: `time`

(F401)